### ISSUE TITLE
.o files and the python executable included in source.

### ISSUE THREAD

**Issue by domdfcoding** [hugovk/python-0.9.1#1]
Hi,

This is fantastic, but I had a bit of trouble building it as the ``src`` directory already contains the ``.o`` files and the final executable created by the compiler.

Did you intend to include these? It builds fine if I delete them and run ``make`` again.

**Comment by hugovk** [comment]
Yes, I intended to include everything in the zip from http://www.dalkescientific.com/writings/diary/archive/2009/03/27/python_0_9_1p1.html with no changes.

See also https://github.com/hugovk/python-0.9.1/blob/master/README.reconstructed

Good to hear it still builds after deleting them!
